### DISCUSSION TITLE
Documents required for calculating cortical thickness

### DISCUSSION THREAD

**Discussion by zhaolei215** [ANTsX/ANTs#1612] [Q&A]
Hello, I would like to ask if I have built a T_template0 based on the my sample, but I don't know how to obtain my own T_ Template0_ BrainCerebellumProbabilityMask.nii.gz and T_ Template0_ BrainCerebellumRegistrationMask. nii.gz These two files. Thanks for your reply.

**Comment by cookpa** [comment]
The brain probability mask can be defined by registration to another template, running a brain-extraction tool like FSL's bet + editing, or by using a deep learning tool like synthstrip, HD-BET, or ANTsPyNet brain extraction. You can optionally smooth the mask to make a fuzzy probability image, or use a tool that gives a probabilistic segmentation.

The registration mask is optional and mostly to save time and mask out regions that can have high variability (eg, the face and neck). You can make one by dilating the brain mask, or just not bother with it depending on how your data is.

**Reply by zhaolei215** [reply]
Thank you for your reply
Can the probabilitymask be understood as first registering with another template to obtain the registered image, and then performing antsBrainExtension. sh on the registered image to obtain probabilitymask. This is a bit more blurry than the mask obtained by directly using antsBrainExtension. sh.
The image I am processing is three-dimensional. I have observed that the registrationmask is indeed larger than the brain contour. I am new fro this field, so I would like to know what specific tool is used to obtain the registrationmask. Thanks!

**Reply by cookpa** [reply]
`antsBrainExtraction.sh` does registration so you can use that directly, using your new template as the input and an existing template to supply the masks.

**Reply by zhaolei215** [reply]
Okay, I will do that. How should I get the registrationmask and what software or tools should I use. Thank you again！

**Reply by cookpa** [reply]
https://figshare.com/articles/dataset/ANTs_ANTsR_Brain_Templates/915436
